.hound.yml

```diff
@@ -0,0 +1,4 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc
+  ignore_file: .eslintignore
```

Comment:
You can add an empty line here for the sake of Standards.

Otherwise, everything else is quite in order.